kayak_core/src/styles/mod.rs

```diff
@@ -139,21 +144,28 @@ define_styles! {
         pub background_color : StyleProp<Color>,
         pub border_color: StyleProp<Color>,
         pub border_radius: StyleProp<(f32, f32, f32, f32)>,
```

Comment:
Should `border_radius` have a similar struct to `Edge` (probably not generic) to provide convenience methods and be more readable (i.e. with obvious top-left, top-right, bottom-left, bottom-right fields)? Maybe `Corner`?